Bug 1210304 - Unused settings has been removed

MariaKolesina opened the pull request:
David's comment:
"Just removing this one select element leave us with an empty panel, doesn't it? I think you should probably remove this entire file and the references to it, right?
And of course you should also remove the homescreen code that reads this grid.layout.cols setting, whereever that code is."
Removed homescreen setting for changing number of columns

MariaKolesina commented:
@harman-red-square/harman-reviewers

ruvsmirnov commented:
:+1: 
Only lint tests must be completed

sergkashin commented:
looks fine to me

AlekoNN commented:
Applying pr_review_passed label, pr_solution_passed label should be set by Egor though.

egorlevichev commented on apps/settings/js/vertical_homescreen.js:
```diff
@@ -1,49 +0,0 @@
-'use strict';
```
As soon as we are removing this module completely, we shall also delete unit test covering it, otherwise we will get test failed result while landing to Mozilla.

The affected test is located in ./test/unit/vertical_homescreen_test.js

MariaKolesina replied:
removed

egorlevichev commented:
One more point: after removing of this settings there seems to be no need of requesting access to datastore `vertical_preferences_store` in the settings/manifest.webapp file, so I think that you shall update manifest also.

MariaKolesina committed:
Bug 1210304 - Unused settings has been removed

David's comment:
"Just removing this one select element leave us with an empty panel, doesn't it? I think you should probably remove this entire file and the references to it, right?
And of course you should also remove the homescreen code that reads this grid.layout.cols setting, whereever that code is."
Removed homescreen setting for changing number of columns
